scss/components/side-nav.scss

```diff
@@ -105,7 +105,7 @@ $block: #{$fd-namespace}-side-nav;
         &.has-child {
             background-image: url(#{$fd-down-arrow-icon});
             background-repeat: no-repeat;
-            background-position: calc(100% - #{$fd-forms-select-width--background-image}) center;
+            background-position: left calc(100% - 10px) top 15px;
```

Comment:
I see you are trying to make a similar change to the child down arrow icon.  It is better, however, the side nav item itself needs some more right padding as text can bleed over onto the icon (see screenshot).  It should also probably have similar spacing on the right of the icon as the list item has on the left side.
![Screen Shot 2019-06-19 at 10 21 17 AM](https://user-images.githubusercontent.com/11407353/59778829-d5ad1180-927c-11e9-9079-db0a4684a48d.png)